frontend/src/components/compiler/PresetSelect.tsx

```diff
@@ -100,6 +100,11 @@ export const PRESETS = [
         compiler: "mwcc_41_60126",
         opts: "-Cpp_exceptions off -stdinc -nodefaults -fp hard -lang=c++ -inline auto,level=2 -ipa file -O4,s -rtti off -sdata 4 -sdata2 4 -align powerpc -enum int",
     },
+    {
+        name: "Super Mario Sunshine",
+        compiler: "mwcc_233_163",
+        opts: " -lang=c++ -Cpp_exceptions off -fp hard -O4 -nodefaults -msgstyle gcc -enum int -rostr",
```

Comment:
Remove this leading space
```suggestion
        opts: "-lang=c++ -Cpp_exceptions off -fp hard -O4 -nodefaults -msgstyle gcc -enum int -rostr",
```